packages/boxel/addon/components/boxel/cover-art/usage.gts

```diff
@@ -0,0 +1,94 @@
+import OntheBrinkofHappinessThumb from '@cardstack/boxel/usage-support/images/cover-art/On-the-Brink-of-Happiness.jpg';
+import ZigandZagThumb from '@cardstack/boxel/usage-support/images/cover-art/Zig-and-Zag.jpg';
+import AllAbouttheQualityofLifeThumb from '@cardstack/boxel/usage-support/images/cover-art/All-About-the-Quality-of-Life.jpg';
+import LoveConquersAllThumb from '@cardstack/boxel/usage-support/images/cover-art/Love-Conquers-All.jpg';
+import MakeMagicThumb from '@cardstack/boxel/usage-support/images/cover-art/Make-Magic.jpg';
+import GoodTimesThumb from '@cardstack/boxel/usage-support/images/cover-art/Good-Times.jpg';
+import MoreThanWeKnowThumb from '@cardstack/boxel/usage-support/images/cover-art/More-Than-We-Know.jpg';
+import NativeArray from '@ember/array/-private/native-array';
+
+export default class CoverArtUsage extends Component {
+  @tracked size = 80;
+  @tracked maxWidth = 190;
+  @tracked spacingMultiplier = 1;
+  @tracked coverThumbs: NativeArray<string> = A([
+    LovetheLoveThumb,
+    HomeIsntSweetThumb,
+    NeverLonelyThumb,
+    OntheBrinkofHappinessThumb,
+    ZigandZagThumb,
+    AllAbouttheQualityofLifeThumb,
+    LoveConquersAllThumb,
+    MakeMagicThumb,
+    GoodTimesThumb,
+    MoreThanWeKnowThumb,
+  ]);
+
+  @action updateCoverThumbs(covers: NativeArray<string>): void {
+    this.coverThumbs = covers;
+  }
+
+  <template>
+    <FreestyleUsage @name="CoverArt">
+      <:example>
+        <BoxelCoverArt
+          @size={{this.size}}
+          @maxWidth={{this.maxWidth}}
+          @covers={{this.coverThumbs}}
+          @spacingMultiplier={{this.spacingMultiplier}}
+        />
+      </:example>
+
+      <:api as |Args|>
+        <Args.Number
+          @name="size"
+          @defaultValue={{80}}
+          @min={{20}}
+          @max={{200}}
+          @description="the size of the largest art, in px. Defaults to 80"
+          @value={{this.size}}
+          @onInput={{fn (mut this.size)}}
+        />
+        <Args.Number
+          @name="maxWidth"
+          @defaultValue={{190}}
+          @min={{100}}
+          @max={{500}}
+          @step={{5}}
+          @description="the maximum width of the layout"
+          @value={{this.maxWidth}}
+          @onInput={{fn (mut this.maxWidth)}}
+        />
+        <Args.Array
+          @name="covers"
+          @type="String"
+          @required={{true}}
+          @description="a collection of thumbnails to lay out. There is no reason to pass more than 5 as they are barely rendered"
+          @items={{this.coverThumbs}}
+          @onChange={{this.updateCoverThumbs}}
+        />
+        <Args.Number
+          @name="spacingMultiplier"
+          @defaultValue={{1}}
+          @min={{0}}
+          @max={{1.5}}
+          @step={{0.1}}
+          @description="Optional adjustment for how closely the covers are displayed"
+          @value={{this.spacingMultiplier}}
+          @onInput={{fn (mut this.spacingMultiplier)}}
+        />
+      </:api>
```

Comment:
I can’t see why but in the preview deployment this slider shows the value below:

<img width="1210" alt="Boxel Components 2022-09-26 15-35-48" src="https://user-images.githubusercontent.com/43280/192375035-158be0ff-c966-4583-a001-d9ad7b03e956.png">

Reply:
This was missing an actual default value I believe, which I specified, so that must be the cause.